Odata-docs/client/v7/getting-started.md

```diff
@@ -141,5 +141,5 @@ And when you run it, you should see the following output:
 
 As you can see, the OData Connected Service generates C# classes corresponding to your specific OData service and uses the underlying OData client library to serialize and deserialize to and from JSON, saving you a lot of work.
 
-In this tutorial you have learned how to use the OData Connected Service to generate client code to interact with a specific OData service. To learn more about about using the client to read and write to the service, check out the [Basic CRUD operations](./basic-crud-operations.md) tutorial.
+In this tutorial you have learned how to use the OData Connected Service to generate client code to interact with a specific OData service. To learn more about about using the client to read and write to the service, check out the [Basic CRUD Operations](/odata/client/basic-crud-operations.md) tutorial.
```

Comment:
@KenitoInc Having the `.md` on the link I believe is reason for this:
![image](https://user-images.githubusercontent.com/28291332/96081063-29575b80-0ec1-11eb-8845-9016dda2df19.png)
You can always click [View] from this table to verify the link open as expected
![image](https://user-images.githubusercontent.com/28291332/96081178-6de2f700-0ec1-11eb-95dd-36b5b08b28a2.png)

Reply:
#Resolved